model-optimizer/extensions/back/ReverseInputChannels.py

```diff
@@ -50,10 +50,24 @@ class InsertReverseChannels(BackReplacementPattern):
     """
     enabled = False
 
+    @staticmethod
+    def get_index_for_api_2_0(node: Node, idx: int):
+        if node.has_valid('rt_info'):
+            rt_info = node.rt_info
+            if rt_info.contains('old_api_map'):
+                old_api_map_version = rt_info.get_attribute_version('old_api_map')
+                old_api_map = rt_info.info['old_api_map', old_api_map_version]
+                if 'inverse_order' in old_api_map.info:
+                    order = old_api_map.info['inverse_order']
+                    assert len(order) > idx >= 0, 'Channel index is incompatible with old_api_map: {}.'.format(idx)
```

Comment:
Could you add node name/id in assert message to detect root cause faster?

Reply:
Done.